src/app/libraries/http_lib.php

```diff
@@ -1,14 +1,19 @@
 <?php
 
 class http_lib extends Library {
-
     function response_code($code, $view_and_exit = true, $view_name = false) {
         http_response_code($code);
         if ($view_and_exit) {
             if (empty($view_name)) {
                 $view_name = strval($code);
             }
-            $this->load_view($view_name);
+            // Check if a dedicated view exists; if not, load common error view
+            $dedicatedView = $this->load_view($code);
+            if (! $dedicatedView) {
+                $this->load_view('http_error', [
```

Comment:
I don't like to call it `http_error`. :/
It is supposed to be generic. We can, in theory, and 200 with this method. The chances of us dog that are very less, so think about it. 
I would prefer views like 4xx, 5xx etc. and also don't see any problem in having 2 different views for server errors and http errors.